config/setupcfg.py

```diff
@@ -72,6 +74,18 @@ def setup_logging(name, level):
     logging.setup(CONF, name)
     LOG = logging.getLogger(None)
 
+    # Add option to do Log Rotation
+    handler = RotatingFileHandler('/etc/hpedockerplugin/3pardcv.log',
+                                  maxBytes=10000000, backupCount=100)
+    formatter = log1.Formatter('%(asctime)-12s [%(levelname)s] '
```

Comment:
we can write this as logging.Formatter instead.

Reply:
oslo_log.log does'nt have Formatter , hence this alias `log1`